src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -45,33 +50,59 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
   const hasErrors = typeof errors !== 'undefined';
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
     const changedVal = e.currentTarget.value;
     setTouched?.(true);
     onChangeHandler(changedVal);
     onValidateHandler?.(setErrors, changedVal);
   };
 
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const handleMouseDownPassword = (e: ClickEvent) => {
+    e.preventDefault();
+  };
+
   return (
     <Box>
-      <PasswordInput
-        label={label + (required ? ' *' : '')}
-        tooltipLabel={
-          (isHidden: boolean) => (isHidden ? getTranslation(translations, 'show') : getTranslation(translations, 'hide'))
-        }
-        value={value}
-        name={name}
+      <InputLabel htmlFor={useOid(name)}>{label + (required ? ' *' : '')}</InputLabel>
```

Comment:
Originally, it was only like this because the PasswordInput component didn't allow the required attr. Now we can get rid of it.
```suggestion
      <InputLabel htmlFor={name} required={required}>{label}</InputLabel>
```